install/tools/istio_vm_setup.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+#
+# Copyright 2017 Istio Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+# Script to install istio components for the raw VM.
+
+# Environment variable pointing to the generated Istio configs and binaries.
+ISTIO_FILES=${ISTIO_FILES:-.}
+
+# Configure network for istio use, using DNSMasq.
+# Will use the generated "kubedns" file.
+function istioNetworkInit() {
+  if [[ ! -r /usr/sbin/dnsmasq ]] ; then
+    apt-get update
+    sudo apt-get -y install dnsmasq
+  fi
+
+  # Copy config files for DNS
+  chmod go+r ${ISTIO_FILES}/kubedns
+  cp ${ISTIO_FILES}/kubedns /etc/dnsmasq.d
```

Comment:
k rename to ISTIO_STAGE_DIR or some such, it's not FILES

Reply:
Done